www-cronrobot/app/javascript/stylesheets/application.scss

```diff
@@ -592,4 +592,28 @@ nav {
     // #pricing { 
     //     min-height: 70vh;
     // }
+}
+
+// Error Styling
+
+.error-grid {
+    display: grid;
+    width: 100%;
+    grid-gap: 20px;
+    color: white;
+    position: -webkit-sticky; /* Safari */
+    position: sticky;
+    top: 0;
+}
+
+.alert-error {
```

Comment:
these colors should be used in application.scss and dashboard.scss

so they should be defined in another scss and included